### ISSUE TITLE
Issue in tutorial2_RestingSource_Group

### ISSUE THREAD

**Issue by armenbagdasarov** [plus-microstate/toolbox#7]
Hello, 

I am going through the tutorial and have a problem with Tutorial 2. See screenshot below. 

![Screen Shot 2022-06-14 at 3 14 05 PM](https://user-images.githubusercontent.com/69819451/173670307-133c043e-eacf-41e7-a2cb-614b577ed2c0.png)

Everything is properly in the path, it just can't load the file. 

Thanks!
Armen

**Comment by lukewtait** [comment]
Hi Armen, 

Could you try running `microstate.functions.install()` in the Matlab command line, and see if that fixes the problem please? Typically this error occurs when the .mat files that are used in the tutorials have not been installed. 

Others have also had this error (assuming this is the problem!), so on the next release I will aim to try and include some sort of check or implement clearer error messages or similar to try and make the solution clearer. 

Please let me know whether that fixes it or not!

Best,
Luke

**Comment by armenbagdasarov** [comment]
Hi Luke, 

When I try running that function, I get **Current version (v1.6) up to date...** in the Command Window. 

But I am still getting the same error when the script tries to load cluster_globalkoptimum_output.mat.

Best, 
Armen

**Comment by lukewtait** [comment]
Ok, no problem.

When you run the tutorial, do you get a pop-up saying the data for tutorial 2 has now been downloaded, and asking if you want to download it now? If so, did you click yes? 

For now, you should hopefully be able to fix this by manually downloading the file from [here](https://github.com/plus-microstate/toolbox/blob/master/tutorials/tutorial2_RestingSource_Group/cluster_globalkoptimum_output.mat) by clicking the download button on that page, and making sure to move the downloaded file to the tutorial 2 folder. Please let me know if that still doesn't work. 

Obviously this isn't an ideal long-term solution, so I won't close this issue just yet. I think I have an idea already how to fix this more permenantly, but I'll do some testing and check.

**Comment by armenbagdasarov** [comment]
Hi Luke, 

I did in fact get the pop-up and pressed yes. The file was also very clearly in the folder so I know that it had been downloaded. 

I just replaced the file  with the one I downloaded manually and it fixed the problem (the name was exactly the same). So, I'm not entirely sure why it didn't work the first time. Perhaps it wasn't downloading properly from the pop-up? 

Thanks!
Armen

**Comment by lukewtait** [comment]
No problem. I believe the error is related to the way the files are stored/downloaded. All .mat files are stored using GitHub large file storage (LFS), which means the file that comes included when you download +microstate from GitHub is actually just a pointer to the actual file stored on LFS which uses the same name and suffix, but isn't a true Matlab .mat file. When you manually download the file individually, it automatically downloads the full file from LFS, so that is why my suggested solution worked. One of the key functions of `microstate.functions.install()` and `check_for_data()` are to save you from downloading them individually - they search for these pointers and download the full file from LFS, but something has obviously failed for this particular file - most likely I made a mistake somewhere in the command for downloading the file (or I just overlooked that file entirely!)

On another note, I should highlight that if you are working through tutorial 2 then the instructions for running `ms.networks_wpli` and MVPA are now outdated, and an update for the tutorial is coming soon. The whole analysis can now be performed using `[p,confusionmat,nets]=ms.networks_wpli(...)`, which both calculates networks and runs the MVPA. The actual tutorial should run fine I think, but the description for generating the nets is outdated. Sorry for this - I'm aiming for another version very soon which should update the tutorial! I'll let you know when it is available.

**Comment by armenbagdasarov** [comment]
This makes sense! I think I figured out one potential reason: When clicking the yes button to install the tutorial data, there is no way to know when the data is done being downloaded. For example, nothing is printed out in the Command Window (e.g., no progress bar), and with large files it may take 5 minutes or it may take 30 minutes depending on the machine. So, perhaps I didn't wait long enough.

**Comment by lukewtait** [comment]
Yes agreed, updating the `check_for_data` function in tutorial 2 has now been put on my to-do list before the next release. Glad the temporary fix is working for you - don't hesitate to ask if you have further questions!

I'll leave this issue "Open" for now, until a more permanent solution is implemented.
